src/seedu/addressbook/AddressBook.java

```diff
@@ -450,8 +450,8 @@ private static String getMessageForSuccessfulAddPerson(String[] addedPerson) {
      * @return feedback display message for the operation result
      */
     private static String executeFindPersons(String commandArgs) {
-        final Set<String> keywords = extractKeywordsFromFindPersonArgs(commandArgs);
-        final ArrayList<String[]> personsFound = getPersonsWithNameContainingAnyKeyword(keywords);
+        final Set<String> keywords = extractKeywordsFromFindPersonArgsInLowerCase(commandArgs);
+        final ArrayList<String[]> personsFound = getPersonsWithNameContainingAnyCaseInsensitiveKeyword(keywords);
```

Comment:
Glad to see that conscious effort to make the code readable =).
Though for `getPersonsWithNameContainingAnyCaseInsensitiveKeyword` can you think of any other way to shorten the method name yet retain its meaning?
Do you think that `getPersonsWithNameContainingAnyCaseKeyword` conveys the same idea?

Furthermore method comments should also be updated.